src/pages/home/ReportScreen.js

```diff
@@ -183,8 +195,10 @@ class ReportScreen extends React.Component {
                     nativeID={CONST.REPORT.DROP_NATIVE_ID}
                     style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
                 >
-                    <FullScreenLoadingIndicator visible={this.shouldShowLoader()} />
-                    {!this.shouldShowLoader() && (
+                    {
+                      (this.state.isGhostScreenVisible || this.shouldShowLoader()) && <ChatGhostUI />
+                    }
+                    {!this.state.isGhostScreenVisible && !this.shouldShowLoader() && (
```

Comment:
Don't you think this `this.state.isGhostScreenVisible` should be completely moved to `this.shouldShowLoader()`.

Reply:
It is WIP

Reply:
+1

**suggestion:** Let's try to just have one boolean that tells us to load or not load the `ReportActionsView` or skeleton UI.